src/main/scala/analysis.scala

```diff
@@ -165,11 +165,24 @@ case object analysis {
       override
       val sqsInitialTimeout =
         10.hours
+
+      override
+      val terminationConfig = TerminationConfig(
+        // if true loquat will terminate after solving all initial tasks
+        terminateAfterInitialDataMappings = true,
+        // if true loquat will terminate after errorQueue will contain more
+        // unique messages than threshold
+        errorsThreshold = Some(5),
+         // maximum time for processing one task
+        taskProcessingTimeout = None,
+        // maximum time for everything
+        globalTimeout = Some(6.hours)
+      )
     }
 
     val defaultConfig =
       AnalysisConfig(
-        loquatName    = "data-analysis",
+        loquatName    = s"data-analysis-${System.currentTimeMillis.toString}",
```

Comment:
This may work fine, but beware ☝️ that Amazon resource names are limited in length 🌵

Reply:
Thanks, I didn't know! But the name is 27 characters long right now (and it will be that long until [November 20th 2286](https://stackoverflow.com/a/9149289)) I googled it quickly and I think the maximum is 64 characters, so it is good for now.

Reply:
Let the future generations deal with this :trollface: